package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "camelcase-keys",
-  "version": "4.1.0",
+  "version": "4.1.1",
```

Comment:
Don't bump version in a pull request. It's up to the maintainers when, how, and what version to bump.